pillar.example

```diff
@@ -3,6 +3,10 @@ hana:
   # If set to false, this packages must be installed before the formula
   # execution manually
   # install_packages: true
+
+  # saptune solution to apply to all nodes ( by default nothing is applied)
```

Comment:
Add the example of how to use `saptune_solution` in a node entry. And besides that, write a warning message that only a unique saptune solution can exist in a node, and if they are different, the formula will fail

Reply:
ok done